home/components/UserReviewSection.tsx

```diff
@@ -20,7 +20,7 @@ export default function UserReviewSection({ snacks, apps }: Props) {
     });
   const theme = useExpoTheme();
 
-  if (!isDevice || !shouldShowReviewSection) {
+  if ((!isDevice && !__DEV__) || !shouldShowReviewSection) {
```

Comment:
No reason for not showing this when running dev mode on the simulator

Reply:
this conditional is a bit confusing to me.

> No reason for not showing this when running dev mode on the simulator

so, "we should show this when running dev mode on the simulator"?

Reply:
for clarity it might be worth splitting this out into two conditionals

```
// Don't render anything when using a simulator in a release build
if (!isDevice && !__DEV__) {
  return null;
}

// Maybe some info here
if (!shouldShowReviewSection) {
  return null;
}
```

Reply:
we can follow up with that separately if you want! i'll merge this and release

Reply:
Yeah, that's definitely easier to read, the idea for showing this when running dev mode on the simulator is to ensure that we can develop/test this feature without requiring us to use a real device